README.md

```diff
@@ -87,10 +87,10 @@ local server = luatest.Server:new({
     workdir = '/path/to/test/workdir',
     -- passed as TARANTOOL_HTTP_PORT, used in http_request
     http_port = 8080,
-    -- passed as TARANTOOL_HTTP_PORT, used in connect_console
-    console_port = 3030,
-    -- passed to net_box.connect in connect_console
-    console_credentials = {user = 'username', password = 'secret'},
+    -- passed as TARANTOOL_HTTP_PORT, used in connect_net_box
+    net_box_port = 3030,
```

Comment:
Probably this is not TARANTOOL_HTTP_PORT

Reply:
fixed